src/runtime/process.rs

```diff
@@ -4,10 +4,10 @@ use crate::{
 };
 use derive_builder::Builder;
 use getset::{CopyGetters, Getters, MutGetters, Setters};
-use once_cell::sync::Lazy;
 use regex::Regex;
 use serde::{de, Deserialize, Deserializer, Serialize};
 use std::path::PathBuf;
+use std::sync::LazyLock;
```

Comment:
```
use of unstable library feature 'lazy_cell'
```

Our dependencies needs to work with stable rust, is that possible with that crate?